src/callback/class-run-sniffer-callback.php

```diff
@@ -349,73 +349,76 @@ function( $standard ) use ( $standards ) {
 			$selected_standards
 		);
 
-		$theme     = \wp_get_theme( $theme_slug );
-		$php_files = $theme->get_files( 'php', 4, false );
+		$args = [];
 
 		// Current theme text domain.
-		$args[ self::TEXT_DOMAINS ][] = $theme_slug;
+		$args[ self::TEXT_DOMAINS ] = [ $theme_slug ];
 
-		// Frameworks.
-		foreach ( $php_files as $key => $file ) {
-			if ( strrpos( $key, 'hybrid.php' ) ) {
-				$args[ self::TEXT_DOMAINS ][] = 'hybrid-core';
-			}
-			if ( strrpos( $key, 'kirki.php' ) ) {
-				$args[ self::TEXT_DOMAINS ][] = 'kirki';
-			}
-		}
+		$all_files = [ 'php' ];
 
-		if ( $check_php_only ) {
-			$all_files = $theme->get_files( [ 'php' ], -1, false );
-		} else {
-			$all_files = $theme->get_files( [ 'php', 'css,', 'js' ], -1, false );
+		if ( ! $check_php_only ) {
+			$all_files = array_merge( $all_files, [ 'css', 'js' ] );
 		}
 
-		$removed_files = [];
+		$theme     = wp_get_theme( $theme_slug );
+		$all_files = $theme->get_files( $all_files, -1, false );
 
 		/**
 		 * Check if a file is minified.
 		 * Loops through all the files and checks if it's minified. If it is, skip it
 		 * because it will break phpcs.
 		 */
 		foreach ( $all_files as $file_name => $file_path ) {
-			// Check if files have .min in the file name.
-			if ( strpos( $file_name, '.min.js' ) !== false || strpos( $file_name, '.min.css' ) !== false ) {
-				unset( $all_files[ $file_name ] );
-				$removed_files[] = $file_name;
-				break;
+
+			// Check for Frameworks.
+			$allowed_frameworks = [
+				'kirki'       => 'kirki.php',
+				'hybrid-core' => 'hybrid.php',
+			];
+
+			foreach ( $allowed_frameworks as $framework_textdomain => $identifier ) {
+				if ( false !== strrpos( $file_name, $identifier ) && ! in_array( $framework_textdomain, $args[ self::TEXT_DOMAINS ], true ) ) {
```

Comment:
Can you just revert this to non-Yoda conditions? I have excluded them in the `phpcs.xml.dist` of the plugin. I really don't like them, and I think they are making code harder to read 😬 

Also we have sniffs in place that should scream if assignment happens in the conditionals 😄

Reply:
ah yeah i noticed that, just a habit i guess :P   i pushed up a change with them as non yoda